src/ifp/src/InitFloorplan.tcl

```diff
@@ -174,6 +174,114 @@ proc initialize_floorplan { args } {
   }
 }
 
+sta::define_cmd_args "make_rows" {
```

Comment:
This looks to be nearly a copy of initialize_floorplan.  Why not just make a -rows_only option rather than a whole new command?

Reply:
I thought about it, but it felt like the current solution is a better separation of responsibilities. Keeps the command and code more focused.

Reply:
It leads to a lot of duplication of code which is a negative.  If you prefer this path then please refactor the tcl to share more.

Reply:
I can just turn intialize_floorplan into a thing that calls make_die and make_rows. Is that your preference?

Reply:
@gadfort any opinion?

Reply:
I think because of the high number of shared flags between the two commands, it would make sense to just add a flag to init_floorplan, that will avoid the two commands getting out of sync, and avoid adding a new command

Reply:
But they really don't share that many flags. I think if I break this up into two commands it'll be cleaner for everyone. No one really needs to know make die even exists.

Reply:
These are all shared
```
  -site site_name
  [-additional_sites site_names]
  [-flip_sites site_names]
  [-row_parity NONE|EVEN|ODD]
```

They will know the make_rows exists and it is a lot of copying imo.  Any new flag added to make_rows will also have to be added to initialize_floorplan and be passed through.  I don't find that cleaner vs a simple flag to skip a step.

Reply:
Let me try anyways, and I think you'll change your mind.

Reply:
I think what I ended up with is pretty clean. I think it's a better organization for ifp, and shouldn't be too much harder to add new args to either command.